### DISCUSSION TITLE
postal not running on defined port 587

### DISCUSSION THREAD

**Discussion by rccess** [postalserver/postal#2927] [Help with using Postal]
i have setup postal and postal is successfully running but smtp port is not showing connected 

<img width="534" alt="image" src="https://github.com/postalserver/postal/assets/32220165/84df9510-bc12-42a5-9bf0-44d42954bdac">
here is postal configuration 

<img width="407" alt="image" src="https://github.com/postalserver/postal/assets/32220165/497509d5-8dda-4cef-acfc-cc94d5f84e6b">

**Comment by willpower232** [comment]
Your screenshot shows that the SMTP container is still restarting so it must be struggling to bind to the port. If you look at its logs, that should give you a clue what is happening.

**Comment by rccess** [comment]
getting this error in logs

<img width="534" alt="image" src="https://github.com/postalserver/postal/assets/32220165/1cd42368-90f2-4329-a4f2-642a3ecd8f74">

<img width="424" alt="image" src="https://github.com/postalserver/postal/assets/32220165/9733eddb-22bf-4d53-b8a3-0153daaee803">

please advise

**Reply by willpower232** [reply]
I'm not sure what that means but I know that port 587 expects TLS by default which is something Postal doesn't do. Does it work if you set the port to 25?

**Reply by rccess** [reply]
i have tried with all port 25, 587, 2525 , same issue 

FYI
i have tried to disable ipv6 from /etc/sysctl.conf but postal was connected on tcp6 only

<img width="306" alt="image" src="https://github.com/postalserver/postal/assets/32220165/5c8324dd-ba7f-4717-abfd-47436307ea13">
and when tried to disable IPV6 permanent through grab
FROM:
GRUB_CMDLINE_LINUX_DEFAULT=""
TO:
GRUB_CMDLINE_LINUX_DEFAULT="ipv6.disable=1"

it is giving the error of 

 /opt/postal/app/app/lib/smtp_server/server.rb:76:in `initialize': Address family not supported by protocol - socket(2) for "::" port 2525 (Errno::EAFNOSUPPORT)

what may be the possible solution to disable ipv6

**Reply by rccess** [reply]
where is this file in docker version

<img width="497" alt="image" src="https://github.com/postalserver/postal/assets/32220165/c7bd25b6-9840-444a-b232-301d1a3dc56a">

https://github.com/atech/postal/blob/master/lib/postal/smtp_server/server.rb

**Reply by willpower232** [reply]
what happens if you specify `default_bind_address: 0.0.0.0`?

**Comment by mostafijar** [comment]
https://www.youtube.com/watch?v=F5x5NxUU5Pc
postal with TLS
